src/App.js

```diff
@@ -1,24 +1,11 @@
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/Calculator';
 import './App.scss';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit, and save to reload.
-          <code>src/App.js</code>
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App flex-center-center">
+      <Calculator />
```

Comment:
- [ ] The App.js component should be class-based.

Reply:
I am sorry I don't understand your review, could you explain, please?